Sources/Puree/Filter.swift

```diff
@@ -24,6 +32,8 @@ public protocol Filter {
     var tagPattern: TagPattern { get }
 
     func convertToLogs(_ payload: [String: Any]?, tag: String, captured: String?, logger: Logger) -> Set<LogEntry>
+}
 
-    init(tagPattern: TagPattern, options: FilterOptions?)
+public protocol InstantiatableFilter: Filter {
+    init(tagPattern: TagPattern)
```

Comment:
please keep the old interface `init(tagPattern: TagPattern, options: FilterOptions?)` for backward compatibility.
I think it is better to migrate just by replacing `Filter` with `InstantiatableFilter`